src/test/java/hw3/Pages/HomePage/HomePage.java

```diff
@@ -0,0 +1,146 @@
+package hw3.Pages.HomePage;
+
+import hw3.AbstractBaseClasses.AbstractBasePage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.openqa.selenium.support.How.CSS;
+import static org.openqa.selenium.support.How.ID;
+
+public class HomePage extends AbstractBasePage {
+
+    public HomePage(WebDriver driver) {
+        super(driver);
+    }
+
+
+    @FindBy(id = "user-icon")
+    private WebElement userIcon;
+
+    @FindBy(how = CSS, using = "#name")
+    private WebElement usernameTextField;
+
+    @FindBy(id = "password")
+    private WebElement passwordTextField;
+
+    @FindBy(how = ID, using = "login-button")
+    private WebElement loginButton;
+
+    @FindBy(id = "user-name")
+    private WebElement usernameLabel;
+
+    @FindBy(xpath = "//ul[@class = 'uui-navigation nav navbar-nav m-l8']/li/a")
```

Comment:
I suppose that here `//ul[@class = 'uui-navigation nav navbar-nav m-l8']` could be use only one class